presenter/loginPresenter.vue

```diff
@@ -0,0 +1,92 @@
+<script
+    setup
+    lang="tsx">
+    
+    import LoginView
+    from "~/views/loginView.vue";
+    import SignupView
+    from "~/views/signUpView.vue";
+
+    import { signIn, signOutUser, signUp } from "~/services/auth";
+
+    const items = [{
+      key: 'login',
+      label: 'Log in'
+    }, {
+      key: 'signup',
+      label: 'Sign up'
+    }]
+
+    const isOpen = ref(false)
+    
+    const auth = useFirebaseAuth()!
+    const user = useCurrentUser()
+
+    function login(username: string, password: string) {
+      signIn(auth, username, password)
+      .then(() => {
+        console.log('Successfully logged in')
+        message.value = 'Successfully logged in'
+        isOpen.value = true
+      })
+      .catch((reason) => {
+        console.error('Failed log in: ', reason)
+        message.value = 'Failed log in: '+reason
+        isOpen.value = true
+      })
+    }
```

Comment:
You always set isOpen.value to true and never use that boolean value, what is its use ? 
also why is message.value needed, I never see you use it

Reply:
It's used in the `UModal` (line 87) to display the message